test/org/sagebionetworks/bridge/validators/StudyValidatorTest.java

```diff
@@ -230,12 +230,6 @@ public void userProfileAttributesCannotBeEmpty() {
         assertValidatorMessage(INSTANCE, study, "userProfileAttributes", "'' must contain only digits, letters, underscores and dashes, and cannot start with a dash");
     }
     
-    @Test
-    public void requiresMissingConsentNotificationEmail() {
-        study.setConsentNotificationEmail(null);
-        assertValidatorMessage(INSTANCE, study, "consentNotificationEmail", "is required");
-    }
```

Comment:
Should this test instead verify that a null consent notification email is valid?

Reply:
Added a test to verify it can be left blank